jpegenc/subblocks/huffman/ac_cr_rom.py

```diff
@@ -0,0 +1,49 @@
+"""MyHDL implementation of AC Chrominance ROM"""
+
+from myhdl import Signal, always, always_comb
+from myhdl import block, intbv, concat
+from .tablebuilder import build_huffman_rom_tables
+
+
+@block
+def ac_cr_rom(clock, address1, address2, data_out_size, data_out_code):
+    """build ac ROM for chrominance"""
+
+    code, size = build_huffman_rom_tables(
+        '../jpegenc/subblocks/huffman/ac_cr_rom.csv')
```

Comment:
this path seems odd, there is a way to find the absolute path of the python module (__file__ or something).  Then you can load the tables from the same directory that the python module exists regardless of how it is installed.  

I looks like right now this only works when executed out of the test directory.